listenbrainz/webserver/static/js/src/stats/UserEntityChart.tsx

```diff
@@ -373,10 +360,22 @@ export default class UserEntityChart extends React.Component<
     window.history.pushState(
       null,
       "",
-      `?page=${page}&range=${range}&entity=${entity}`
+      this.buildURLParams(page, range, entity)
     );
   };
 
+  buildURLParams = (
+    page: number,
+    range: UserStatsAPIRange,
+    entity: Entity
+  ): string => {
+    return new URLSearchParams({
```

Comment:
URLSearchParams isn't supported in IE, and honestly this method is huge... Happy to revert it back to string interpolation like the setURLParams method used to be if we want to make this shorter...

Reply:
Good question. I can't really think of many arguments one way or the other.
I guess if URLSearchParams doesn't bring any extra functionality and has worse browser support we should probably revert to string interpolation, or if we think there are potential benefits to URLSearchParams we can check for support and use our previous version as a fallback (`if(window.URLSearchParams){…} else {//template string instead}`)

Reply:
I'll switch it back to a string interpolation, which takes this method back down to 1 line of code.